getting-started/GettingStarted.md

```diff
@@ -0,0 +1,48 @@
+# Getting Started: Publishing a Kubernetes App on Azure Marketplace
+
+### Introduction
+Thank you for joining us in our Kubernetes Apps public preview program. We’re delighted to have you participate in our vibrant and ever-growing ecosystem, where you can publish and monetize your Kubernetes application. This document has all the necessary steps to get you started and can enable you to publish your application in 1-2 weeks to the Kubernetes marketplace in Azure.
+
+### Pre-requisites
+You must meet the following criteria to be able to participate in the public preview:
+* The Kubernetes application should have a Helm-chart based application that is free of malware. We highly recommend doing a [pre-scan and fixing the vulnerabilities](https://github.com/bobmital/kubernetes-offer-samples/blob/05611a5c6485db980fe27fb8999af728382b547d/getting-started/Vulnerability-Scan.md) before you push the application through the pipeline.
+* You should have an active publishing tenant or access to a [Partner Center account](https://learn.microsoft.com/en-us/partner-center/overview).
+* You should be able to adhere to several of our supported [billing models (licensing options)](https://learn.microsoft.com/en-us/partner-center/marketplace/marketplace-containers). 
+* Must be actively engaged.
+
+### ISV expectations
+If the necessary steps are followed, you can publish the application within 1-2 weeks! We expect you to be actively engaged and contact us if you face any necessary roadblocks after following the required steps. We appreciate your participation in our program. However, if we notice that your engagement levels have decreased, we may need to give priority to other participants due to the high volume of interest we receive.
```

Comment:
Can ISV expectation content be in email?

Reply:
Rephrased it and added it to the prerequisites section instead. It can be in email.